docs/sources/configure-client/grafana-agent/go_pull.md

```diff
@@ -1,258 +1,224 @@
-| enabled  | boolean | Enable this profile type to be scraped.    | true                    | no       |
-| path     | string  | The path to the profile type on the target. | "/debug/pprof/goroutine" | no       |
-| delta    | boolean | Whether to scrape the profile as a delta.  | false                   | no       |
-
-### `profile.mutex`
-
-Collects profiles on mutexes.
-
-| Argument | Type    | Description                                | Default             | Required |
-| -------- | ------- | ------------------------------------------ | ------------------- | -------- |
-| enabled  | boolean | Enable this profile type to be scraped.    | true                | no       |
-| path     | string  | The path to the profile type on the target. | "/debug/pprof/mutex" | no       |
-| delta    | boolean | Whether to scrape the profile as a delta.  | false               | no       |
-
-### `profile.process_cpu`
-
-Collects profiles on CPU consumption for the process.
-
-| Argument | Type    | Description                                | Default                | Required |
-| -------- | ------- | ------------------------------------------ | ---------------------- | -------- |
-| enabled  | boolean | Enable this profile type to be scraped.    | true                   | no       |
-| path     | string  | The path to the profile type on the target. | "/debug/pprof/profile" | no       |
-| delta    | boolean | Whether to scrape the profile as a delta.  | true                   | no       |
-
-### `profile.fgprof`
-
-Collects profiles from an fgprof endpoint.
-
-| Argument | Type    | Description                                | Default           | Required |
-| -------- | ------- | ------------------------------------------ | ----------------- | -------- |
-| enabled  | boolean | Enable this profile type to be scraped.    | false             | no       |
-| path     | string  | The path to the profile type on the target. | "/debug/fgprof"   | no       |
-| delta    | boolean | Whether to scrape the profile as a delta.  | true              | no       |
-
-### `profile.godeltaprof_memory`
-
-Collects profiles from godeltaprof memory endpoint. The delta is computed on the target.
-
-| Argument | Type    | Description                                | Default                  | Required |
-| -------- | ------- | ------------------------------------------ | ------------------------ | -------- |
-| enabled  | boolean | Enable this profile type to be scraped.    | false                    | no       |
-| path     | string  | The path to the profile type on the target. | "/debug/pprof/delta_heap" | no       |
-
-### `profile.godeltaprof_mutex`
+### Prepare Grafana Agent Flow configuration file
 
-Collects profiles from godeltaprof mutex endpoint. The delta is computed on the target.
+In the Grafana Agent Flow configuration file, you need to add at least two blocks: `pyroscope.write`
+and `pyroscope.scrape`.
```

Comment:
Should we mention the location and name of the configuration file? Or a default location or file name? (Understanding that this may vary depending upon the operating system.)

Reply:
location and file name does not matter, it can be /foo/bar239.png